R/check_data_regeneration.R

```diff
@@ -48,6 +49,8 @@ check_data_regeneration <- function(database, forest_reserve = "all") {
     query_database(database, query_regeneration, selection = selection)
 
   incorrect_regeneration <- data_regeneration %>%
+    filter(survey_reg == 10) %>%
+    select(-survey_reg) %>%
```

Comment:
En nog eentje.  (Eigenlijk gaat dat wel gemakkelijk, door als reviewer in GitHub onder het tabje 'Files changed' naar beneden te scrollen.  Ik zal er enkele laten staan, dan kan je dit ook eens bekijken en weet je de volgende keer waar zoeken als je het lokaal niet vindt.)

```suggestion
    select(-"survey_reg") %>%
```